packages/optimizer/README.md

```diff
@@ -119,8 +119,8 @@ Example:
 ```js
 const ampOptimizer = AmpOptimizer.create({
   extensionVersions: {
-    "amp-twitter": "0.1"
```

Comment:
Are all the style changes on purpose? Does this repo have a default prettier configuration / lint-check that should be handling these?